ApiDoctor.Validation/DocFile.cs

```diff
@@ -477,7 +477,7 @@ protected bool ParseMarkdownBlocks(IssueLogger issues)
                         var previousBlock = this.OriginalMarkdownBlocks[previousBlockIndex];
                         if (null != previousBlock && previousBlock.BlockType != BlockType.html)
                         {
-                            issues.Error(ValidationErrorCode.MissingMetadataBlock,
+                            issues.Warning(ValidationErrorCode.MissingMetadataBlock,
```

Comment:
Does this change make it so that if the metadata block is missing, the checkin won't be blocked, but will raise a warning to the PR submitter? Still, I'd like to see tests for these validation rules since this message is a call to action to the PR submitter to add the metadata block.

Reply:
This new validation rule generates approximately 490 errors in the docs which will need to be fixed before these feature goes live, to give them time to update and fix the docs, we agreed to turn it into a warning for now.

Reply:
we still fail the build on warnings, though. at least we should. since warnings are just as serious as errors for most of the issues raised by the tools. there is no meaningful distinction between warning and error.

Reply:
Yes. We do fail a build on error but not on warnings (--ignore-warnings) flag.

Reply:
@BarryShehadeh as noted in your comment below, We do fail on u

> The only caveat with this that it's time extensive to enforce, and having it as a Warning doesn't really do us any good. Because it will be ignored in MS Graph Docs, but we do NOT* ignore warning in our API docs. If the graph team still see this as relevant requirement, then we probably should have it as an optional setting.

Considering how many errors this will generate, we can close it. When I did the PR, it generated 490 errors.